data/ecosystem/vendors.yaml

```diff
@@ -414,6 +414,12 @@
   contact: https://github.com/adnanrahic
   oss: true
   commercial: true
+- name: Alauda
+  nativeOTLP: true
+  url: https://www.alauda.io/
```

Comment:
can you point to a page of your documentation or website that highlights how you make use of opentelemetry and how end users can send data to your backend via OTLP?

Reply:
thanks